src/Microsoft.DotNet.Interactive.Formatting.Tests/HtmlFormatterTests.cs

```diff
@@ -305,8 +305,23 @@ is a
                           $"{PlainTextBegin}{instance.HtmlEncode()}{PlainTextEnd}");
             }
 
+            [Fact]
+            public void HtmlFormatter_returns_plain_for_BigInteger()
+            {
+                var formatter = HtmlFormatter.GetPreferredFormatterFor(typeof(BigInteger));
+
+                var writer = new StringWriter();
+
+                var instance = BigInteger.Parse("78923589327589332402359");
 
+                formatter.Format(instance, writer);
+
+                writer.ToString()
+                    .Should()
+                    .Be($"{PlainTextBegin}78923589327589332402359{PlainTextEnd}");
```

Comment:
Adding to [comment](https://github.com/dotnet/interactive/pull/1246/files#r613817544). Here, if it was via `FormatTo`,  it would return the result without `PlainTextBegin`. Now it seems to be working:
![image](https://user-images.githubusercontent.com/31178401/114830638-19b7c180-9dd5-11eb-8728-1f307c2341bb.png)